docs/manual/developer_guide.adoc

```diff
@@ -73,13 +73,14 @@ Advantage of this solution is that you will not need to change parameters during
 
 **Required boot parameters:**
 
-```
+----
 inst.updates=http://gateway:8000/update.img
-```
+----
 
 Here, `gateway` is supposed to refer to your host machine.
-When setting the NAT network in the VM setup, `gateway` will be recognized correctly (at least on RHEL7).
-In other cases, input your IP address.
+When setting the NAT network in the VM setup, `gateway` will be recognized correctly.
+This works if the `dnsmasq` is set up correctly, which is the case on RHEL systems, but not on Fedora.
+In other cases, input the IP address of the VM.
```

Comment:
This is a little bit confusing. The only way I got it to work on Fedora was by selecting "network install" method and I used the address of my virtual bridge interface (`virbr0`) instead of `gateway` (as by default the python3 http.server binds to all interfaces). Then in "Network selection" part I selected the "Specify shared device name" and I put `virbr0` there. Maybe we can document this approach for Fedora use case.

Reply:
Yes, I have rephrased this.